lsp-rust.el

```diff
@@ -587,6 +587,21 @@ The command should include `--message=format=json` or similar option."
 
 (defvar-local lsp-rust-analyzer-inlay-hints-timer nil)
 
+(defface lsp-rust-analyzer-inlay-type-face
```

Comment:
Shouldn't these inherit `lsp-rust-analyzer-inlay-face` instead of defining a completely new one?
So user will not stumped upon breaking visual change when update `lsp-mode`

Reply:
Maybe we should also document how to change faces. This is a common problem for new emacs users.